### PR TITLE
WT-8274 Run function.py on all our source files

### PR DESCRIPTION
**Pull request by jiechenbo**
This ticket aims running function.py on all c source files which also includes test source files and adding all the comments that was missing a comment. Originally the missing_comment() function would be performed on wiredtiger specific source files.

### CODE REVIEW

File: src/utilities/util_alter.c [hunk 1]
```diff
@@ -8,13 +8,21 @@
 
 #include "util.h"
 
+/*
+ * usage --
```
Comment (jiechenbo) [inline src/utilities/util_alter.c:12]: I added a TODO as part of the boilerplate comment. What do you guys think of this?

**Review by keithbostic — APPROVED** [review]
lgtm.

One thought, do we want to require a comment on `main()` functions?

Offhand, I'd probably leave it as it is and require comments for `main()`, mostly because it's simpler to not have the special case, but I wanted to call it out, because I think most people don't normally comment `main()`.

File: src/utilities/util_alter.c (hunk 1 above)
Comment (keithbostic) [reply src/utilities/util_alter.c:12]: I think we should have a TODO there. Or, alternatively, a FIXME.

**Review by etienneptl — APPROVED** [review]
LGTM! Hopefully we start updating each missing description soon!

**Review by fionaochee — APPROVED** [review]
LGTM!

**Pull request merged**